extensions/amp-list/0.1/amp-list.js

```diff
@@ -66,6 +97,7 @@ export class AmpList extends AMP.BaseElement {
    * @private
    */
   rendered_(elements) {
+    removeChildren(this.container_);
```

Comment:
Doesn't matter for screen-readers, `element` is probably better since it would allow page authors to override it (maybe they want aria-live='assertive' for certain lists).